<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HS: Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectAll">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>55</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deselectAll">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deselect all</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QListWidget" name="animeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="loadingStatus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="selectQuality">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
